sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientTestBase.java

```diff
@@ -209,4 +202,23 @@ public void sleep(long millis) {
             e.printStackTrace();
         }
     }
+
+    /**
+     * Returns a stream of arguments that includes all combinations of eligible {@link HttpClient HttpClients} and
+     * service versions that should be tested.
+     *
+     * @return A stream of HttpClient and service version combinations to test.
+     */
+    protected Stream<Arguments> getTestParameters() {
+        // when this issues is closed, the newer version of junit will have better support for
+        // cartesian product of arguments - https://github.com/junit-team/junit5/issues/1427
+        List<Arguments> argumentsList = new ArrayList<>();
+        getHttpClients()
+            .forEach(httpClient -> {
+                for (KeyServiceVersion serviceVersion : KeyServiceVersion.values()) {
```

Comment:
Since this is cryptography client test, should this be CryptographServiceVersion?

Reply:
This is left over. I forgot to remove. 
All tests here are using the helper method under TestHelper.

Removed